src/admin/content-block/content-block.types.ts

```diff
@@ -16,6 +18,10 @@ export type ButtonTypeOptions = 'primary' | 'secondary';
 
 export type WidthOptions = 'full-width' | '500px' | '400px';
 
+export type ImageGridFillOptions = 'cover' | 'contain' | 'auto';
+
+export type ImageGridTextAlignOptions = 'left' | 'center' | 'right';
```

Comment:
Ik zou dit aanpassen naar `AlignOptions` of `AlignOption` want deze wordt niet enkel hier gebruikt.